openreview/journal/process/preferred_emails_process.py

```diff
@@ -0,0 +1,50 @@
+def process(client, invitation):
+
+    journal = openreview.journal.Journal()
+
+    print('Get profiles for all the reviewers')
+    reviewers = client.get_group(journal.get_reviewers_id()).members
+
+    print('Get profiles for all the action editors')
+    action_editors = client.get_group(journal.get_action_editors_id()).members
+
+    print('Get profiles for all the assigned reviewers and action editors')
+    groups = client.get_all_groups(prefix=journal.venue_id + '/Paper')
+
+    for group in groups:
+        if '/Reviewer_' in group.id:
+            reviewers += group.members
+        elif '/Action_Editor_' in group.id:
+            action_editors += group.members
```

Comment:
Not all reviewers and action editors are in the reviewers and action editors group?

Reply:
nop, some reviewers can be invited to review one paper but they are not official reviewers (they don't belong to the venue reviewers group)

Reply:
Would it be necessary to remove duplicates?

Reply:
I'm doing that in line 20 when getting the profiles.